simulator/build.gradle.kts

```diff
@@ -91,3 +91,29 @@ tasks.register<Copy>("untarTestBlockStream") {
 tasks.named("processResources") { dependsOn(tasks.named("untarTestBlockStream")) }
 
 tasks.named("sourcesJar") { dependsOn(tasks.named("untarTestBlockStream")) }
+
+// Task to prepare and run Docker container
+tasks.register("startSimulatorDockerContainers") {
```

Comment:
The name of the task should be matching with the one in the server `startDockerContainer` as is per the [agreed upon convention](https://github.com/hashgraph/hedera-block-node/discussions/264#discussioncomment-11272142).

Reply:
Yes, completly forgot about it, will change it to match 👍